packages/manager/modules/manager-pci-common/src/api/data/volume.ts

```diff
@@ -0,0 +1,24 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+
+export type TVolume = {
+  id: string;
+  attachedTo: string[];
+  creationDate: string;
+  name: string;
+  description: string;
+  size: number;
+  status: string;
+  statusGroup: string;
+  region: string;
+  bootable: boolean;
+  planCode: string;
+  type: string;
+  regionName: string;
+};
+
+export const getVolumes = async (projectId: string): Promise<TVolume[]> => {
+  const { data } = await v6.get<TVolume[]>(
+    `/cloud/project/${projectId}/volume`,
```

Comment:
You need to use the regionalized version, this one will be deprecated

https://eu.api.ovh.com/console/?section=%2Fcloud&branch=v1#get-/cloud/project/-serviceName-/region/-regionName-/volume

Reply:
we are currently using the same request as used in production, your request is out of the scope of this PR